packages/tests/server/react/invalidateQueries.test.tsx

```diff
@@ -221,11 +222,11 @@ describe('invalidateQueries()', () => {
       'invalidate-3-all',
       'invalidate-4-predicate',
     ]) {
-      // click button to invalidate
-      utils.getByTestId(testId).click();
+      await waitFor(async () => {
+        // click button to invalidate
+        await userEvent.click(utils.getByTestId(testId));
```

Comment:
shouldn't this be outside the `waitFor`?

Reply:
Test failed on the outside

Reply:
fixed the test